views/layout.html

```diff
@@ -100,45 +100,63 @@
   </div>
 <script>
 (function() {
+    const update_search = (search_element, should_delay, debug_label) => {
```

Comment:
This looks intentionally identical to the code in the other layout file. Please add a comment to each noting that these should stay in sync.